### PR TITLE
Update Issue templates

### PR DESCRIPTION
**Pull request by kk5dire**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
 labels: enhancement, Feature
 assignees: ''
 
 ---
 
-**Is your feature request related to a problem? Is it additive or related to the current content?**
+`***[ Begin placeholder text! ]***`
+ 1. Change the title to fit your suggestion (Tool or Suggestion)
+    - 1. Are you requesting a new tool? Then pick *tool!*
+    - 2. Or are you requesting an enchancement to an already there tool. Then pick *suggestion!*
```
Comment (benfoster04) [inline .github/ISSUE_TEMPLATE/enhancement.md:13]: "to an already there tool" -> "to a pre-existing tool"

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
+ 
+`***[ End placeholder text! ]***`
+ 
+**Summary**
 
+A brief idea of what you want!
+ 
+**Description**
+
+A more detailed in-depth description of what you want.
```
Comment (benfoster04) [inline .github/ISSUE_TEMPLATE/enhancement.md:31]: Across the file really, should opt not to use instances of "what you want" in favour of "what you're suggesting", "what you're recommending", or "what you would like"

**Review by benfoster04 — CHANGES_REQUESTED** [review]
Apart from changes requested, LGTM and thank you for the work :)

File: .github/ISSUE_TEMPLATE/enhancement.md (hunk 2 above)
Comment (kk5dire) [reply .github/ISSUE_TEMPLATE/enhancement.md:31]: Text content was there prior to review. However, noted and signed.

**Review by Tech-TTGames — APPROVED** [review]
Looks good!

**Review by Tech-TTGames — APPROVED** [review]
That was a missclick.

**Review by benfoster04 — APPROVED** [review]
Fabulous, thank you for the changes! LGTM, cleared for merge.

**Pull request merged**